src/UndockedRegFreeWinRT/Nuget/Microsoft.Windows.UndockedRegFreeWinrt.nuspec

```diff
@@ -7,7 +7,7 @@
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Regfree WinRT enables using WinRT types without the registry</description>
+    <description>Regfree WinRT enables using WinRT types from unpackaged applications</description>
```

Comment:
should this be more descriptive to call out that this is backfilling functionality that's now inbox?

Reply:
"Regfree WinRT enables using WinRT types from unpackaged applications, a feature that is available starting on Windows version 1903 and above. This nuget package backfills this functionality to previous Windows versions down to RS2."

How's this?